components/Buttons/src/ShapeThemer/MDCFloatingButtonShapeThemer.h

```diff
@@ -24,12 +24,13 @@
 @interface MDCFloatingButtonShapeThemer : NSObject
 
 /**
- Applies a shape scheme's properties to an MDCFloatingButton.
+ Applies a rounded rectangular shape to an MDCFloatingButton instance. Each corner has a radius of
+ 50% the FAB's height.
 
- @param shapeScheme The shape scheme to apply to the component instance.
+ @param shapeScheme This parameter is ignored.
```

Comment:
Please explain here why it is ignored, see the comment in the code: "This is an override of the default scheme to fit the baseline values."

Reply:
our FAB in the baseline is an override, that's how the shape scheme works in the baseline. If a user wants to create a diamond shaped FAB he should create his own themer that receives the shape scheme value and applies it on the FAB.

Reply:
Ok, then I think this change makes sense and later we can remove the parameter from the themer entirely.  Updating the documentation is a non-breaking change and is a more accurate reflection of what the code should be doing. 

I'm not sure what you would like me to do to further explain why the shape is ignored. I think the method docs do a pretty good job of explaining the shape that will be produced when the themer is applied. Given that the shape is always a "rounded rectangle", documenting the parameter as ignored/unused aligns with the method docs.   

If there is something missing or you have copyedit suggestions, please feel free to put them up in a PR after this and I'd be happy to approve. Since this isn't a blocking review, I hope you also feel that these changes are at least better than what was there previously.